core/src/main/java/org/pentaho/di/core/vfs/KettleVFS.java

```diff
@@ -129,12 +131,23 @@ public static FileObject getFileObject( String vfsFilename, VariableSpace space,
       // If not, we are going to assume it's a file.
       //
       boolean relativeFilename = true;
-      String[] schemes = fsManager.getSchemes();
-      for ( int i = 0; i < schemes.length && relativeFilename; i++ ) {
-        if ( vfsFilename.startsWith( schemes[ i ] + ":" ) ) {
+      String[] initialSchemes = fsManager.getSchemes();
+
+      relativeFilename = checkForScheme( initialSchemes, relativeFilename, vfsFilename, space, fsOptions );
```

Comment:
What does `relativeFilename` mean?  Looks like checkForScheme returns false if the files scheme is present, not sure what makes that "relative".  Also, why false?
Maybe some comments or javadoc could clarify.